src/index.html

```diff
@@ -11,7 +11,174 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
+    <form
+      action="http://localhost:8080"
+      method="post"
+    >
+      <fieldset>
+        <legend>Personal information</legend>
+
+        <label class="form-field">
+         Surname:
+          <input
+            type="text"
+            name="surname"
+            autocomplete="off"
+            required
+            >
```

Comment:
```suggestion
          <input
            type="text"
            name="surname"
            autocomplete="off"
            required
          >
```
[check checklist item 3](https://github.com/mate-academy/layout_html-form/blob/master/checklist.md) and fix it all over your code